### DISCUSSION TITLE
Configuring Category for workload

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#104] [Q&A] [answered]
Hi Team, 
As we can see in the below image, upon clicking new item there are few workloads under "Others" category. Can you please let us know where we can configure to add our workload under a specific category. Please do find the attached image for reference regarding category.

<img width="536" alt="Sample_Workload_Item_Image" src="https://github.com/user-attachments/assets/382cfb7f-6aa4-49f9-b3d6-0781a4e340c8">

Thank you

**Comment by IdanGoldfarb1989** [comment, marked as answer]
Hi, we just added support to integrate with the categories to our [sample](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/d28f8e89b5a88dfe61a97fee3d67ba5e3b0487b4/Frontend/Package/Item1.json#L70).


See [schema definition](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/d28f8e89b5a88dfe61a97fee3d67ba5e3b0487b4/Frontend/validation/Item.schema.json#L144) for new property
